MAUI/TreeMap/migration.md

```diff
@@ -0,0 +1,1450 @@
+---
+layout: post
+title: Migrate from Xamarin to .NET MAUI TreeMap (SfTreeMap) | Syncfusion
+description: Learn about Migrating from Syncfusion Xamarin.Forms TreeMap to .NET MAUI TreeMap.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+ 
+# Migrate from Xamarin.Forms TreeMap to .NET MAUI TreeMap (SfTreeMap)
+
+To make the migration from the [Xamarin SfTreeMap](https://www.syncfusion.com/xamarin-ui-controls/xamarin-treemap) to the `.NET MAUI SfTreeMap` easier, most of the APIs from the `Xamarin SfTreeMap` were kept in the `.NET MAUI SfTreeMap`. However, to maintain the consistency of API naming in the `.NET MAUI SfTreeMap`, some of the APIs have been renamed. Please find the difference in the following topics.
+
+## Namespaces
+
+<table>
+<tr>
+<th>Xamarin SfTreeMap</th>
+<th>.NET MAUI SfTreeMap</th></tr>
+<tr>
+<td>Syncfusion.Xamarin.SfTreeMap</td>
+<td>Syncfusion.Maui.TreeMap</td></tr>
+</table>
+
+## Initialize Control
+
+To initialize the control, import the treeMap control namespace and initialize the `SfTreeMap` as shown in the following code sample.
+
+<table>
+<tr>
+<th>Xamarin SfTreeMap control</th>
+<th>.NET MAUI SfTreeMap control</th></tr>
+<tr>
+<td>
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:treemap="clr-namespace:Syncfusion.SfTreeMap.XForms;assembly=Syncfusion.SfTreeMap.XForms">
+     <treemap:SfTreeMap x:Name="treeMap"/>
+</ContentPage>
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.XForms.TreeMap;
+...
+
+SfTreeMap treeMap = new SfTreeMap();
+this.Content = treeMap;
+
+{% endhighlight %}
+{% endtabs %}
+
+</td>
+<td>
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage 
+...
+xmlns:treemap="clr-namespace:Syncfusion.Maui.TreeMap;assembly=Syncfusion.Maui.TreeMap">
+    <treemap:SfTreeMap x:Name="treeMap"/>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.TreeMap;
+…
+
+SfTreeMap treeMap = new SfTreeMap();
+this.Content = treeMap;
+
+{% endhighlight %}
+{% endtabs %}
+</td></tr>
+</table>
+
+## Classes 
+
+<table>
+<tr>
+<th>Xamarin SfTreeMap control</th>
+<th>.NET MAUI SfTreeMap control</th>
+<th>Description</th>
+</tr>
+
+<tr>
+<td>{{'[SfTreeMap](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.SfTreeMap.html)'| markdownify }}</td>
+<td>SfTreeMap</td>
+<td>The SfTreeMap control that allows you to visually represent hierarchical data with nested rectangles, conveying quantitative information through variations in size and color.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>TreeMapItemInfo</td>
+<td>Contains item info for the selected item.</td>
```

Comment:
Add ToolTiptemplate, leafItemTemplate - We are binding DataContent for these properties as well.

Reply:
ToolTiptemplate, leafItemTemplate is not necessary as it is related to Classes